Added an admin setting to enable/disable viewing deprecation messages

mwesten opened the pull request:
Hi Huub,
Added an admin setting to suppress the deprecation warnings.
(accidentally committed to my master instead of the 'prepare next version' branch. Pull request is made to the correct branch.
<img width="1402" height="237" alt="librewolf_JrSNff8diP" src="https://github.com/user-attachments/assets/09434220-0acc-4e0e-b6f7-cfdc146895d4" />

mwesten committed:
Revert "Added an admin setting to enable/disable viewing deprecation messages in debug output."

This reverts commit f15c0662ab9657d52f53233cddd0cd76beddf4a7.

mwesten committed:
Added an admin setting to enable/disable viewing deprecation messages
in debug output.

HuubMons commented:
There are language conflicts. At this moment I don't know how to solve that, I have to check how to do that.

mwesten commented:
Hmmmm sorry....
Seems that .po and .mo files have a tendency to do that... (and the conflicts can't be resolved in the web-merge editor, but need a git client, like git-desktop or sourcetree)
What's easiest for you ? Just add the translation lines in the php code and leave the po generation with poedit to you ? (and add the translations to the pull-request as text ?)

HuubMons commented:
Yes, just skip the language files. I will rebuild them using poedit.

mwesten committed:
Added debug deprecation setting

Added an admin setting to enable/disable viewing deprecation messages in debug output.
(now left out the admin files)

mwesten commented:
Hi Huub,
Reverted the commit and committed again without the language-files.
Merging the pull request shouldn't be a problem anymore.

(Don't you forget to revert the nl.po language deletion you did?)

The translations that need to be added are:

en:
```php
msgid "Show deprecated messages in debug output"
msgstr "Show deprecated messages in debug output"

msgid "This shows all deprecation messages when debug is on"
msgstr "This shows all deprecation messages when debug is on"
``` 


nl:
```php
msgid "Show deprecated messages in debug output"
msgstr "Toon meldingen van verouderde code in debug regels"

msgid "This shows all deprecation messages when debug is on"
msgstr "Dit laat meldingen zien van alle verouderde code, wanneer Debug aan staat"
```

HuubMons commented:
Ha, dat is gelukt. Bedankt voor de update!

De vertalingen doe ik later nog (ben wat druk momenteel).
